src/asmcnc/comms/model_manager.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+ROOT_DIR = os.path.dirname(os.path.dirname(os.getcwd()))
+DWT_FILE_PATH = os.path.join(ROOT_DIR, "dwt.txt")
+
+PLYMOUTH_SPLASH_FILE_PATH = "/usr/share/plymouth/debian-logo.png"
+
+SKAVA_UI_IMAGES_PATH = os.path.join(os.getcwd(), "asmcnc", "skavaUI", "img")
+YETI_SPLASH_PATH = os.path.join(SKAVA_UI_IMAGES_PATH, "yeti_splash_screen.png")
+DWT_SPLASH_PATH = os.path.join(SKAVA_UI_IMAGES_PATH, "dwt_splash_screen.png")
+
+
+def is_machine_drywall():
+    # () -> bool
+    """
+    Checks for the existence of the dwt.txt file.
+    :return: True if the file exists, False otherwise.
+    """
+    return os.path.exists(DWT_FILE_PATH)
+
+
+def set_machine_drywall(is_drywall):
```

Comment:
What's your idea when should this be called? I assume sometime in final test?

Reply:
Yeah, I thought it could be called from I assume the button that final test operator would press

Reply:
Yes it's tested on console XL. I don't have a 7 inch console right now, but it's the same.

Reply:
I did sudo reboot and it actually shows the updated splash screen on shutdown as well as startup.